Platform/TigerlakeBoardPkg/Script/StitchIfwiConfig_tglh.py

```diff
@@ -0,0 +1,238 @@
+#  This is an IFWI stitch config script for Slim Bootloader
+#
+# Copyright (c) 2020, Intel Corporation. All rights reserved. <BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+extra_usage_txt = \
+"""This is an IFWI stitch config script for Slim Bootloader For the FIT tool and
+stitching ingredients listed in step 2 below, please contact your Intel representative.
+
+  1. Create a stitching workspace directory. The paths mentioned below are all
+     relative to it.
+
+  2. Extract required tools and ingredients to stitching workspace.
+     - FIT tool
+       Copy 'fit.exe' or 'fit' and 'vsccommn.bin' to 'Fit' folder
+     - BPMGEN2 Tool
+       Copy the contents of the tool to Bpmgen2 folder
+       Rename the bpmgen2 parameter to bpmgen2.params if its name is not this name.
+      - Components
+       Copy 'cse_image.bin'     to 'Input/cse_image.bin'
+       Copy PMC firmware image  to 'Input/pmc.bin'.
+       Copy EC firmware image   to 'Input/ec.bin'.
+       copy ECregionpointer.bin to 'Input/ecregionpointer.bin'
+       Copy GBE binary image    to 'Input/gbe.bin'.
+       Copy ACM firmware image  to 'Input/acm.bin'.
+       Copy DiagnosticACM firmware image  to 'Input/DiagnosticAcm.bin'.
+
+
+  3. Openssl
+       Openssl is required for stitch. the stitch tool will search evn OPENSSL_PATH,
+       to find Openssl. If evn OPENSSL_PATH is not found, will find openssl from
+       "C:\\Openssl\\Openssl"
+
+  4. Stitch the final image
+       EX:
+       Assuming stitching workspace is at D:\Stitch and building ifwi for TGL platform
+       To stitch IFWI with SPI QUAD mode and Boot Guard profile VM:
+         StitchIfwi.py -b vm -p tglu -w D:\Stitch -s Stitch_Components.zip -c StitchIfwiConfig.py

```

Comment:
the tglh example use "tglh" instead of "tglu"

Reply:
Thank for the catch, will get that fixed in the next patch.